csvcubed-pmd/csvcubedpmd/urireplacement/urireplacement.py

```diff
@@ -0,0 +1,90 @@
+import click
+import sys
+import os
+
+from chardet.universaldetector import UniversalDetector
+import logging
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
I don't think this logging configuration line should be here. I think it should be in the `main` class - so we configure the logging level once for the whole CLI instead of having different modules competing to alter the logging level.